Dockerfile.desktop

```diff
@@ -26,25 +26,36 @@ ENV DEBIAN_FRONTEND=noninteractive \
 
 # Install requirements to install tools
 RUN dependencies=' \
-        bzip2 \
-        ca-certificates \
-        curl \
-        firefox \
-        python3-pip \
-        python3-setuptools \
-        python3-wheel \
-        sudo \
-        wget \
-        build-essential \
-        gcc \
-        python3 \
-        tox \
-        pipx \
-        libgtk-3-dev \
-        libasound2 \
-        libdbus-glib-1-dev \
-        xvfb \
-        libgl1-mesa-dri \
+    gcc \
+    musl-dev \
+    xvfb \
+    xauth \
+    curl \
+    wget \
+    gnupg2 \
+    libgtk-3-0 \
+    libdbus-glib-1-2 \
+    libxt6 \
+    libx11-xcb1 \
+    libxcomposite1 \
+    libxdamage1 \
+    libxfixes3 \
+    libxrender1 \
+    libxext6 \
+    libxrandr2 \
+    libasound2 \
+    libpango-1.0-0 \
+    libpangocairo-1.0-0 \
+    libdrm2 \
+    libgbm1 \
+    libatspi2.0-0 \
+    bzip2 \
+    libglib2.0-0 \
+    libnss3 \
+    libgconf-2-4 \
+    libfontconfig1 \
+    libdbus-glib-1-2 \
+    xz-utils \
```

Comment:
Any chance we can a-Z these? 🙏 
The teeniest of nits, but also, sorting locally told me that `libdbus-glib-1-2` is duped (L37 and L57).